Common/include/CConfig.hpp

```diff
@@ -805,8 +805,9 @@ class CConfig {
   Gas_Constant,         /*!< \brief Specific gas constant. */
   Gas_ConstantND,       /*!< \brief Non-dimensional specific gas constant. */
   *Molecular_Weight;    /*!< \brief Molecular weight of an incompressible ideal gas (g/mol). */
-  unsigned short nMolecular_Weight; /*!< \brief Number of species molecular weights. */
-  su2double Specific_Heat_Cp, /*!< \brief Specific heat at constant pressure. */
+  unsigned short nMolecular_Weight, /*!< \brief Number of species molecular weights. */
+  nSpecific_Heat_Cp;              /*!< \brief Number of species specific heat constants at constant pressure. */
+  su2double *Specific_Heat_Cp, /*!< \brief Specific heat at constant pressure. */
   Specific_Heat_CpND,         /*!< \brief Non-dimensional specific heat at constant pressure. */
```

Comment:
Should not be needed now
```suggestion
```

Reply:
If I remove this variable, I will have to remove the function 

void SetSpecific_Heat_CpND(su2double val_specific_heat_cpnd) { Specific_Heat_CpND = val_specific_heat_cpnd; }

and the line in the CIncEulerSolver.cpp where it is used

 Specific_Heat_CpND  = config->GetSpecific_Heat_Cp()/Gas_Constant_Ref;          config->SetSpecific_Heat_CpND(Specific_Heat_CpND);

Should I do this in this pull request?

Thank you so much in advance

Reply:
Yes, let's try not to leave loose ends that could be confusing in the future.
Yes remove that line, and `Specific_Heat_CpND = config->GetSpecific_Heat_Cp()/Gas_Constant_Ref;` should be replaced by `Specific_Heat_CpND = config->GetSpecific_Heat_CpND();`

Reply:
Thanks!! I will do it.